node-pixels.js

```diff
@@ -1,193 +1,200 @@
-'use strict'
+"use strict";
```

Comment:
@MayorScript Usually PRs with large formatting overhauls aren't readily accepted. To avoid PR tag, you might have better luck with only swapping `request` for `node-fetch` =/

Reply:
Thanks for the feedback @twolfson ... I'll make the change right away.